pom.xml

```diff
@@ -30,13 +30,25 @@
     </developers>
 
     <properties>
-        <jenkins.version>1.580.1</jenkins.version>
+        <jenkins.version>1.596.1</jenkins.version>
```

Comment:
Yes, that's why I've upgraded to Jenkins > 1.590. It saves me from one extra null check.
Thanks for the tip, when the time comes to upgrade to Jenkins > 1.653, I'll replace `getActiveInstance` with `getInstance`.